pkg/debugapi/peer.go

```diff
@@ -35,21 +35,6 @@ func (s *server) peerConnectHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	err = s.Addressbook.Put(address, addr)
```

Comment:
did we decide to remove this functionality? i'm not following why is this removed